extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -301,12 +288,6 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return this.isAmpAdElement();
   }
 
-  /** @override */
-  delayAdRequestEnabled() {
-    return getExperimentBranch(this.win, RENDER_IDLE_DELAY_REQUEST_EXP) ==
```

Comment:
Just want to doublecheck that the intent here is to fully delete, as opposed to in the adsense impl where it just returns true

Reply:
confirmed